matlab-hello-world-model/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM hello-world:latest
```

Comment:
I'd add a comment here saying this is a local image that has been Matlab-exported as opposed to https://hub.docker.com/_/hello-world